config/prysm/slasher/beacon.yaml

```diff
@@ -19,7 +19,6 @@ monitoring-host: 0.0.0.0
 
 grpc-gateway-host: 0.0.0.0
 grpc-gateway-port: 3500
-grpc-gateway-corsdomain: http://localhost:7500
```

Comment:
"grpc-gateway-corsdomain: http://localhost:7500" is used to connect prysm web interface, without corsdomain web interface is inaccessible

Reply:
right, but that's a validator config (and it's in the validator.yaml).
No meaning in the beacon config

Reply:
Yes you right, it doesn't make sense in beacon config, good PR, thanks again.